apps/web/screens/Article/components/ArticleChatPanel/config.ts

```diff
@@ -12,262 +12,306 @@ export const liveChatIncConfig: Record<string, LiveChatIncChatPanelProps> = {
   },
 }
 
+const setupOneScreenWatsonChatBot = (
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  instance: any,
+  categoryTitle: string,
+  categoryGroup: string,
+) => {
+  if (sessionStorage.getItem(categoryGroup) !== categoryTitle) {
+    sessionStorage.clear()
+  }
+  sessionStorage.setItem(categoryGroup, categoryTitle)
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const preSendhandler = (event: any) => {
+    event.data.context.skills['main skill'].user_defined[
+      `category_${categoryTitle}`
+    ] = true
+  }
+
+  instance.on({ type: 'pre:send', handler: preSendhandler })
+
+  instance.updateHomeScreenConfig({
+    is_on: false,
+  })
+}
+
+enum WatsonIntegration {
+  ASKUR = 'b1a80e76-da12-4333-8872-936b08246eaa',
+  ASKUR_SYSLUMENN = '0c96e8fb-d4dc-420e-97db-18b0f8bb4e3f',
+}
+
+enum WatsonServiceInstance {
+  ASKUR = 'bc3d8312-d862-4750-b8bf-529db282050a',
+}
+
+enum Font {
+  IBM_PLEX_SANS = '"IBM Plex Sans", "Open Sans", Arial, sans-serif',
+}
```

Comment:
Should this belong in a central fonts file?

Reply:
I moved this to the same file as the watson integration id's live